packages/react-portal/src/virtualParent/getVirtualParent.ts

```diff
@@ -0,0 +1,11 @@
+import { isVirtualElement } from './isVirtualElement';
+/**
+ * Gets the virtual parent given the child element, if it exists.
+ */
+export function getVirtualParent(child: HTMLElement): HTMLElement | undefined {
+  let parent: HTMLElement | undefined;
+  if (child && isVirtualElement(child)) {
```

Comment:
Do you need to check `child` here?
It is `HTMLElement` and it's `isVirtualElement`'s responsibility to check it's truthy

Reply:
You're right, it's unnecessary